ocamltest/builtin_actions.ml

```diff
@@ -305,6 +305,13 @@ let has_symlink = make
     "symlinks available"
     "symlinks not available")
 
+let not_root = make
+  ~name:"not_root"
+  ~description:"Pass if the current user isn't root"
```

Comment:
I think the following is slightly clearer:
```suggestion
  ~description:"Skip test if the current user is root"
```

Reply:
done